data/remnant/remnant 2 cognizance.txt

```diff
@@ -97,7 +97,7 @@ mission "Remnant: Cognizance 2"
 				`	"Thanks for the explanation!"`
 					goto acceptance
 				`	"I thought prefects were specialists."`
-			`	"Well, they are and are not. Most prefects are assigned particular tasks that they are expected to devote themselves to. Often, this task relates to their area of expertise, but it is not always the case. Generally speaking, a prefect is considered to be the ultimate authority on tasks that they have been assigned to, but they are expected to take into account the suggestions from other prefects and the specialists in the fields relevant to the task.`
+			`	"Well, they are and are not. Most prefects are assigned particular tasks that they are expected to devote themselves to. Often, this task relates to their area of expertise, but it is not always the case. Generally speaking, a prefect is considered to be the ultimate authority on tasks that they have been assigned to, but they are expected to take into account the suggestions from other prefects and the specialists in the fields relevant to the task."`
```

Comment:
The enthusiasm is awesome, but as Warpcore said, there are some cases where quotation marks are intentionally not placed. This is one such instance. More precisely, this is an example of a paragraph break *within* a continuous block of speech by a single person.